swift/README.md

```diff
@@ -3,14 +3,6 @@
 This folder contains example Ice applications written in Swift. Each example is a simple client-server application that
 demonstrates a specific feature or programming technique.
 
-> Ice for Swift is only supported on macOS and iOS platforms.
```

Comment:
Same here.
All good removing from the README, but I would add this to `NIGHTLY.md` under "Ice for Swift".